packages/cli/src/commands/setup/cache/templates/memcached.ts.template

```diff
@@ -0,0 +1,21 @@
+import { createCache, MemcachedClient } from '@redwoodjs/api/cache'
+
+import { logger } from './logger'
+
+const memJsFormattedLogger = {
```

Comment:
Just curious - how is this different? Should it also be logger.error or just info?

Reply:
The logger object for `memjs` assumes that there be a `log()` function on whatever object you pass in as the logger, and our logger doesn't have that. So this is like a little adapter pattern to make it match up.

Reply:
Gotcha, thanks!